classes/article/ArticleGalleyDAO.inc.php

```diff
@@ -493,15 +493,15 @@ function getByPubIdType($pubIdType, $contextId = null, $title = null, $author =
 			FROM	submission_galleys g
 				' . ($contextId != null?' JOIN submissions s ON (s.submission_id = g.submission_id)':'') . '
 				LEFT JOIN published_submissions ps ON (ps.submission_id = g.submission_id)
-				LEFT JOIN issues i ON (ps.issue_id = i.issue_id AND i.date_published IS NOT NULL)
+				LEFT JOIN issues i ON (ps.issue_id = i.issue_id)
```

Comment:
@asmecher, I will try. Could you also take a look at the changes in the IssueDAO below? There I also had to move i.published at the beginning...